### ISSUE TITLE
Unable to use Nest with pyNN

### ISSUE THREAD

**Issue by ramapati166** [NeuralEnsemble/PyNN#668]
I am running Ubuntu 18.04 LTS
I have successfully installed NEST 2.20.0 with Anaconda Python, by using the following command **"conda create --name ENVNAME -c conda-forge nest-simulator"**  given in the site _**"https://nest-simulator.readthedocs.io/en/stable/installation/index.html**"_,I then installed pyNN using pip.

But, "import pyNN.nest as sim" gives:
```
CSAConnector: libneurosim support not available in NEST.
Falling back on PyNN's default CSAConnector.
Please re-compile NEST using --with-libneurosim=PATH
/home/user/anaconda3/envs/ENVNAME/lib/python3.8/site-packages/pyNN/nest/__init__.py:53: UserWarning:Unable to install NEST extensions. Certain models may not be available.
```
The pyNN installation page(http://neuralensemble.org/docs/PyNN/installation.html) says that this can be resolved by adding **nest-config to PATH** as it didn't work that is I still get this message even after adding the directory containing nest-config to the PATH, I tried **pip uninstall PyNN**, then re-installed with **pip install --no-binary :all: PyNN** which was mentioned these above mentioned site.I have already applied several tips given in internet but didn't get fruitful result.If any one  can help to solve this issue I will be highly obliged to him/her.Thanks in advance.

**Comment by apdavison** [comment]
Hi @ramapati166 

What you see there are only warnings, not errors. They tell you that some of the available _extensions_ to PyNN (accelerated CSAConnector and some additional NEST models) could not be installed. _However_, the standard PyNN functionality should work.

 I have created a new issue, #669, to remind me to make these warnings less confusing.

I should also note that PyNN has not been tested with NEST 2.20.0. It may work, please give it a try, but we've only tested with NEST 2.18.0. Keep an eye on #670 for when NEST 2.20.0 support becomes officially available.

**Comment by dstandage** [comment]
Hi @apdavison - I'm having the same issue as @ramapati166 . It happens with NEST 2.18.0 and 2.20.0. Do you know which NEST models aren't installed - and how to install them if needed?

**Comment by apdavison** [comment]
Hi @dstandage. Currently it is only one model: StochasticTsodyksMarkramSynapse

The problem is perhaps the installation with pip (although I thought that had been fixed). As a possible workaround, uninstall the pip version, then download the package and try to install it with "python setup.py install"

**Comment by dstandage** [comment]
Thanks @apdavison - I wasn't able to get setup.py to work with nest in a conda environment, but it's good to know that I'm only missing one model (though I do like Tsodyks-Markram synapses!). It sounds like the problem may have been fixed for global installations, but not conda envs - I should have mentioned that I'm using conda

**Comment by apdavison** [comment]
This is now fixed, in the upcoming 0.9.6 release. Please comment below if you still encounter problems.
